code/ecdsap256sha2/Hacl.Impl.ECDSA.P256SHA256.Verification.fst

```diff
@@ -409,12 +409,12 @@ val ecdsa_verification_u8: pubKey: lbuffer uint8 (size 64) -> r: lbuffer uint8 (
       )  
     (ensures fun h0 result h1 -> modifies0 h0 h1 /\ 
       (
-	let publicKeyX =  nat_from_bytes_le (as_seq h1 (gsub pubKey (size 0) (size 32))) in 
-	let publicKeyY =  nat_from_bytes_le (as_seq h1 (gsub pubKey (size 32) (size 32))) in 
-	let r = nat_from_bytes_le (as_seq h1 r) in 
-	let s = nat_from_bytes_le (as_seq h1 s) in 
+	let publicKeyX = nat_from_intseq_le (Hacl.Spec.ECDSA.changeEndian (uints_from_bytes_be (as_seq h0 (gsub pubKey (size 0) (size 32))))) in 
```

Comment:
Isn't this just `nat_from_intseq_be (as_seq h0 (gsub pubKey (size 0) (size 32)))`?